server/plugin.go

```diff
@@ -490,3 +540,23 @@ func addFields(fields []*model.SlackAttachmentField, title, msg string, short bo
 		Short: model.SlackCompatibleBool(short),
 	})
 }
+
+func messageToJSON(message string) (bool, []uint8) {
+	messagefields := strings.Split(message, "\n")
+	if len(messagefields) > 0 {
+		//split each line of the cloudformation event message to field and value
+		var fields = make(map[string]string)
+		for _, field := range messagefields[:len(messagefields)-1] {
+			if len(strings.Split(field, "=")) == 2 {
+				fields[strings.Split(field, "=")[0]] = strings.Split(field, "=")[1]
```

Comment:
Something like this may read better:

```suggestion
			parts := strings.Split(field, "=")
			if len(parts) == 2 {
				fields[parts[0]] = parts[1]
```

Reply:
@mickmister Thank you for your feedback! I will update the feature with the new changes.